README.md

```diff
@@ -1,11 +1,89 @@
-Silvercar Javascript Style Guide
----
+# JS Constable <!-- omit in toc -->
+
+> Javascript/Typescript rules to ~~code~~ live by
+
+![JS Constable logo](https://i.imgur.com/lJ4lgLD.png)
```

Comment:
We should probably be using an s3 for this asset, not imgur.

Reply:
We will also get a more fun logo. When ready I can put it on s3

Reply:
Hosting within repo, as suggested by @brycehendrix on #guild-devops